doc/lsst.pipe.base/creating-a-pipelinetask.rst

```diff
@@ -472,7 +473,12 @@ passed on to the run method. The body of the run method checks if the background
 has been passed, and if so adds it back in and does a local background
 subtraction.
 
-To bring this all together, see :ref:`pipeline-appendix-c`
+To bring this all together, see :ref:`pipeline-appendix-c`.
+
+Note that the same mechanism works for output connections:
+an output from a `PipelineTask` can be made optional (under config control)
+or a config switch can even be used to choose between different ways of
+providing output.
```

Comment:
Might be worth clarifying that this is different from making an input optional at runtime by setting `minimum=0` in the `Input` connection declaration.  In the configuration case, the connection is treated as if it never existed by QG generation and execution code, while `minimum=0` allows a quantum to be generated and run even no dataset for that input are found.  (There is no `minimum` for `Output` connections because they are _always_ considered runtime-optional.)

Reply:
Let me know if you think I've captured it now.